features/admin.branding.v1/components/branding-page-layout.tsx

```diff
@@ -263,14 +264,28 @@ const BrandingPageLayout: FunctionComponent<BrandingPageLayoutInterface> = (
                                                 value={ brandingMode }
                                                 disabled={ isBrandingAppsRedirect }
                                             >
-                                                <ToggleButton value={ BrandingModes.ORGANIZATION }>
+                                                <ToggleButton
+                                                    value={ BrandingModes.ORGANIZATION }
+                                                    onClick={ () => {
+                                                        updateActiveTab(
+                                                            BrandingPreferencesConstants.TABS.GENERAL_TAB_ID
+                                                        );
+                                                    } }
+                                                >
                                                     <BuildingIcon
                                                         className="toggle-button-icon"
                                                         size={ 14 }
                                                     />
                                                     { t("extensions:develop.branding.pageHeader.organization") }
                                                 </ToggleButton>
-                                                <ToggleButton value={ BrandingModes.APPLICATION }>
+                                                <ToggleButton
+                                                    value={ BrandingModes.APPLICATION }
+                                                    onClick={ () => {
+                                                        updateActiveTab(
+                                                            BrandingPreferencesConstants.TABS.GENERAL_TAB_ID
```

Comment:
Shall we add a check here so that this update only happens when the user currently in text branding, otherwise updating the same tab can result in flickering UI.

Reply:
Won't that be inconsistent ?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6648/commits/bf87d0bd08c75bb6c1a74af6090629425bd7326c